scanpy/tools/_louvain.py

```diff
@@ -133,7 +137,22 @@ def louvain(
                 g, partition_type,
                 **partition_kwargs,
             )
-            # adata.uns['louvain_quality'] = part.quality()
+            if 'quality' in dir(part):
+                q = part.quality()
+                if isinstance(part, louvain.RBConfigurationVertexPartition) and q > 1.0: #scale
+                    m = weights.sum() if use_weights else g.ecount()
+                    q /= m if directed else m*2
+                if isinstance(part, (louvain.RBConfigurationVertexPartition,
+                                     louvain.ModularityVertexPartition)):
```

Comment:
please use [black line wrapping](https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines) here and above.

Reply:
Done (I hope)